src/tests/__mocks__/answers.js

```diff
@@ -4,4 +4,6 @@ export default [{
 }, {
   frontEndUniqueKey: 'testID2',
   display_name: 'test display name 1',
+}, {
+  display_name: 'test display name 1',
```

Comment:
Please give these mock answers different display names just to illustrate that they are different answers.

Reply:
I've done that